src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,40 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+import { findCurent } from './helper';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const checkId = (currentTab, tab) => currentTab.id === tab.id;
+  const currentTab = findCurent(tabs, selectedTabId);
+  const selectedTab = (tab) => {
```

Comment:
This name is not perfect - for functions, you should prefer verbs in present form (like "doSomething")
In your case, that should be like `selectTab`, `onTabClick`, etc